src/training/ui/menu.rs

```diff
@@ -114,6 +123,18 @@ unsafe fn render_submenu_page(app: &App, root_pane: &mut Pane) {
                 title_text.text_outline_enable(false);
                 title_text.set_color(178, 199, 211, 255);
             }
+
+            submenu_ids.iter().for_each(|id| {
+                if let Some(icon) = menu_button.find_pane_by_name_recursive(id) {
+                    if id == &submenu.submenu_id {
+                        icon.set_visible(true);
+                    } else {
+                        icon.set_visible(false);
+                    }
+                }
```

Comment:
Two things:
1) The rest of this file uses `.unwrap()` to explicitly panic if we don't find a pane, and I think we should follow that here rather than the ignore paradigm of `if let`. I will create a future PR to have `find_pane_by_name_recursive` panic by default and also a separate one that doesn't in `try_find_pane_by_name_by_recursive`.
2) set_visible(true) and (false) can usually be one-lined:

```suggestion
                menu_button.find_pane_by_name_recursive(id)
                    .unwrap()
                    .set_visible(id == &submenu.submenu_id);
```

Reply:
Thank you for explaining how the codebase works here and why. I understand the pattern better now :)